src/test/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsTest.java

```diff
@@ -0,0 +1,614 @@
+    }
+
+    private void editAbbreviation(Abbreviation testAbbreviation) throws JabRefException {
+        viewModel.abbreviationsNameProperty().set(testAbbreviation.getName());
+        viewModel.abbreviationsAbbreviationProperty().set(testAbbreviation.getAbbreviation());
+        viewModel.editAbbreviation();
+    }
+
+    private void saveFilesToPreferences() {
+        savelyAddNewFileToViewModel(testFile1Entries);
+        savelyAddNewFileToViewModel(testFile3Entries);
+        savelyAddNewFileToViewModel(testFile4Entries);
+        savelyAddNewFileToViewModel(testFile5EntriesWithDuplicate);
+        viewModel.saveExternalFilesList();
+    }
+
+    /**
+     * Select the most recently added journal abbreviation file
+     */
+    private void selectLastJournalFile() {
+        viewModel.changeActiveFile(viewModel.journalFilesProperty().get(viewModel.journalFilesProperty().size() - 1));
+    }
+
+    /**
+     * Select the last abbreviation in the list of abbreviations
+     */
+    private void selectLastAbbreviation() {
+        viewModel.currentAbbreviationProperty()
+                .set(viewModel.abbreviationsProperty().get(viewModel.abbreviationsCountProperty().get() - 1));
+    }
+
+    /**
+     * This will add the given file to the current view model.
+     * In case the given file already exists it will fail the current test.
+     * This is useful for testing purposes so you don't have to always handle the exceptions.
+     *
+     * @param fileToAdd to the view model
+     */
+    private void savelyAddNewFileToViewModel(String path) {
+        try {
+            viewModel.addNewFile(path);
+        } catch (JabRefException e) {
+            Assert.fail("The same file has been opened/created multiple times.");
+        }
+    }
+
+    /**
+     * This will add the given abbreviation to the current view model.
+     * In case the given abbreviation already exists it will fail the current test.
+     * This is useful for testing purposes so you don't have to always handle the exceptions.
+     *
+     * @param testAbbreviation which will be added to the view model
+     */
+    private void savelyAddAbbreviation(Abbreviation testAbbreviation) {
+        try {
+            addAbbrevaition(testAbbreviation);
+        } catch (JabRefException e) {
+            Assert.fail("The same abbreviation already exists in the current list.");
+        }
+    }
+
+    /**
+     * This will edit the given abbreviation.
+     * In case the given abbreviation already exists it will fail the current test.
+     * This is useful for testing purposes so you don't have to always handle the exceptions.
+     *
+     * @param testAbbreviation which will be edited
+     */
+    private void savelyEditAbbreviation(Abbreviation testAbbreviation) {
+        try {
+            editAbbreviation(testAbbreviation);
+        } catch (JabRefException e1) {
+            Assert.fail("The same abbreviation already exists in the current list.");
+        }
```

Comment:
do not catch exception in tests, just throw them. In this case, you hide the exception from JUnit, and the user does not get the stack trace, only your short text.